### DISCUSSION TITLE
Images and GIFs not displaying in GitHub anywhere

### DISCUSSION THREAD

**Discussion by MeetDev2602** [community/community#70451] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The images and gifs are not loaded into any repository on my network and any browser(google chrome, Firefox, opera, etc.).

![image For Reference](https://github.com/community/community/assets/121008158/08ba690e-e560-45f8-8876-8e8943cce102)

![github website inspect this error show](https://github.com/community/community/assets/121008158/23cf2103-9884-4ac8-a802-86fcda20d173)

I have Jio's 5G network and it has upto 50MBPS speed. And the network is never slowing down.

![my network speed](https://github.com/community/community/assets/121008158/8311a28b-0a94-41a0-add0-09337073ebf7)

I have tried opening the page in incognito mode, but it doesn't work.

if i use VPN in my network. so, it works fine. but loading very slow.

**Comment by ghost** [comment]


**Reply by MeetDev2602** [reply]
but this is not my repository another person

**Reply by MeetDev2602** [reply]
if i use VPN in my network. so, it works fine. but loading very slow.

**Reply by krishnarajav** [reply]
I'm facing similar issue. I cannot preview any images in my account. But the same images are visible to others.

**Comment by eklairs** [comment]
The actually reason is that JIO network has blocked loading contents from `raw.githubusercontent.com`

You can see this question for more info - https://github.com/orgs/community/discussions/42655

And solution - https://stackoverflow.com/questions/76292834/jio-blocked-raw-githubusercontent-com

**Comment by iprattham** [comment]
Is it happening with only a specific repository or you're facing this issue all over Github?
For eg, I've 3-4 gifs added in my readme, so are you able to view these?
Link: https://github.com/iprattham/pratham

**Reply by adityasharmaa09** [reply]
Yeah, I'm able to view with wifi or other networks except jio

**Comment by MeetDev2602** [comment]
In Windows, you can achieve the same result by adding an entry to the "hosts" file. However, the process is a bit different. You'll need to open a text editor with administrator privileges to edit the "hosts" file. Here's how you can do it using Command Prompt:

Open Command Prompt as an administrator:

Press Win + S, type "Command Prompt" or "cmd" in the search bar.
Right-click on "Command Prompt" in the search results.
Select "Run as administrator."
Type the following command to open the "hosts" file in Notepad:
```
notepad C:\Windows\System32\drivers\etc\hosts
```
Notepad will open with the "hosts" file. You might see a UAC (User Account Control) prompt; click "Yes" to allow the application to make changes.

In Notepad, add the following line at the end of the file:
```
185.199.108.133 raw.githubusercontent.com
```
Save the file and close Notepad.

Now, the IP address "185.199.108.133" is associated with the hostname "raw.githubusercontent.com" in your Windows "hosts" file.

**Reply by MeetDev2602** [reply]
this worked my network

**Comment by MeetDev2602** [comment]
if Use Linux try this
https://github.com/orgs/community/discussions/42655#discussioncomment-4943564
